src/actions/GenericActions.js

```diff
@@ -38,6 +38,26 @@ export function getData(
     );
 }
 
+/**
+ * 
+ * @param {*} entity - entity should always be `documentView`
+ * @param {*} windowId 
+ * @param {*} viewId 
+ * @param {*} rowId 
+ */
+export function getAttributes(
+    entity, windowId, viewId, rowId
+) {
+    return axios.get(
+        config.API_URL +
+        '/' + entity +
+        '/' + windowId +
+        (viewId ? '/' + viewId : '') +
+        (rowId ? '/' + rowId : '') +
```

Comment:
imho the viewId and rowId shall always be present. if not, there is an error.

Reply:
yes, they will always be presented but left them to make it generic function

Reply:
@teosarca do you want me to update these comments?